arduino/cores/cores.go

```diff
@@ -111,9 +111,10 @@ func (platform *Platform) GetOrCreateRelease(version *semver.Version) (*Platform
 	return release, nil
 }
 
-// GetRelease returns the specified release corresponding the provided version,
+// GetReleaseRelaxedVersion returns the specified release corresponding the provided version,
 // or nil if not found.
-func (platform *Platform) GetRelease(version *semver.Version) *PlatformRelease {
+func (platform *Platform) GetReleaseVersion(version *semver.Version) *PlatformRelease {
```

Comment:
I don't understand this change, may you explain it?

Reply:
Because I suppress the warning othewise I have to replace the type *semver.Version with fmt.Stringer
https://github.com/mvdan/interfacer#suppressing-warnings

Reply:
I see that the method has been renamed from `GetRelease` to `GetReleaseVersion`, what's exactly the warning you suppressed with this change?

Reply:
This is the error form the linter interfacer
```bash
arduino/cores/cores.go:116:38: `version` can be `fmt.Stringer` (interfacer)
func (platform *Platform) GetRelease(version *semver.Version) *PlatformRelease {
```
If you want to remove this error you have to use the interface instead the specific struct.
In this way
```go
func (platform *Platform) GetRelease(version fmt.Stringer) *PlatformRelease {
	return platform.Releases[version.String()]
}
```
Choose the solution that seems better for you, I thought that is necessary use the semver.Version so I prefer to rename the method

Reply:
Ok, now I understand, changing to fmt.Stringer doesn't look like a great improvement to me.

My problem is that `GetReleaseVersion` looks like a method to get the `Version` of a `Platform`, instead this is a method to find a `PlatformRelease` with a specific version. Let's call it `FindReleaseWithVersion`.

Reply:
Done 9ecd7dd688d0af6194971590e9a7598093384bd2